his/what-is-his.md

```diff
@@ -58,11 +58,11 @@ For more information, see [Data Integration (Planning)](core/data-integration-pl
 
 ## Application Integration
 
-Transaction Integrator (TI) allows enterprise developers to call business rules in host mainframe (Customer Information Control System and Information Management System) and midrange (i5/OS) programs using Visual Studio and the .NET Framework. TI is comprised of a Visual Studio plug-in designer, administration tool (TI Configuration Tool) and runtime components that integrate host applications with the .NET Framework. Enterprise developers can define .NET clients using an intuitive COBOL or RPG source code import wizard within the TI designer. 
+Transaction Integrator (TI) allows enterprise developers to call business rules in host mainframe (Customer Information Control System and Information Management System) and midrange (IBM i) programs using Visual Studio and the .NET Framework. TI is comprised of a Visual Studio plug-in designer, administration tool (TI Configuration Tool) and runtime components that integrate host applications with the .NET Framework. Enterprise developers can define .NET clients using an intuitive COBOL or RPG source code import wizard within the TI designer. 
```

Comment:
Why "IBM i" and not "IBM System i"? There's another paragraph below that's the same.

Reply:
It should be IBM System i. That is the official name.

Reply:
I'll leave you to fix.